docs/commands/generate-arazzo.md

```diff
@@ -42,23 +35,88 @@ npx @redocly/cli generate-arazzo <your-OAS-description-file> [-o | --output-file
 
 - -o, --output-file
 - string
-- Path to the OAS description file. If the file name is not provided, the default name is used - `auto-generate.yaml`. Example: `npx @redocly/cli generate-arazzo OAS-file.yaml -o=example.yaml`
+- Path to the OAS description file. If the file name is not provided, the default name is used - `auto-generate.arazzo.yaml`. Example: `npx @redocly/cli generate-arazzo OAS-file.yaml -o=example.arazzo.yaml`
 
----
-
-- --extended
-- boolean
-- By default, data for requests comes from the description at runtime. This option generates a test config file with data populated from the description. Example: `npx @redocly/cli generate-arazzo OAS-file.yaml -o=example.yaml --extended`.
+{% /table %}
 
----
+## Examples
 
-- --with-expectations
-- boolean
-- By default, data for requests comes from the description at runtime. This option generates a test config file with data populated from the description with additional expectations. Example: `npx @redocly/cli generate-arazzo OAS-file.yaml -o=example.yaml --with-expectations`.
+Run the command: `npx @redocly/cli generate-arazzo warp.openapi.yaml`
+
+The command generates a `auto-generate.arazzo.yaml` file in the current directory.
+
+The contents of the generated file are:
+
+```yaml {% title="auto-generate.arazzo.yaml" %}
+arazzo: 1.0.1
+info:
+  title: Warp API
+  version: 1.0.0
+sourceDescriptions:
+  - name: warp.openapi
+    type: openapi
+    url: warp.openapi.yaml
```

Comment:
```suggestion
    url: https://warp-single-sidebar.redocly.app/_spec/apis/index.yaml
```

Reply:
This is a bug then. This content is auto-generated. We should fix it and release again.